filestore/protoboards.go

```diff
@@ -0,0 +1,118 @@
+package filestore
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/influxdata/chronograf"
+)
+
+// ProtoboardExt is the the file extension searched for in the directory for protoboard files
+const ProtoboardExt = ".json"
+
+// Protoboards are instantiable JSON representation of dashbards.  Implements ProtoboardsStore.
+type Protoboards struct {
+	Dir     string                                      // Dir is the directory containing protoboard json definitions
+	Load    func(string) (chronograf.Protoboard, error) // Load receives filename, returns a Protoboard from json file
```

Comment:
Where is this function set?

Reply:
found it :) https://github.com/influxdata/chronograf/pull/4217/files#diff-30ff908306987253d049f1a97bb43216R29

Reply:
what do you mean by "set"? `NewProtoboards` returns a `&Protoboards` with `Load: protoboardLoadFile,`, and `protoboardLoadFile` is defined, if that's what you are asking